docs/distributed/README.md

```diff
@@ -1,74 +1,86 @@
 
-### Data protection
+**Note:** Each cluster must reside in the same network, but multiple clusters from multiple networks can be federated.
 
-Distributed Minio provides protection against multiple node/drive failures and [bit rot](https://github.com/minio/minio/blob/master/docs/erasure/README.md#what-is-bit-rot-protection) using [erasure code](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide). As the minimum disks required for distributed Minio is 4 (same as minimum disks required for erasure coding), erasure code automatically kicks in as you launch distributed Minio.
+### Data Protection
 
-### High availability
+Distributed Minio provides protection against multiple node/drive failures and [bit rot](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide.html#whatisrot) using [erasure code](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide). Since distributed Minio and erasure code both require a minimum of four hard drives, erasure code is automatically used when distributed Minio is launched.
 
-A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
+### High Availability
 
-For example, an 8-node distributed Minio setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+A standalone server running Minio Server will go down if the hard drives go offline. In contrast, a distributed Minio configuration with N disks will keep data safe as long as N/2 or more hard drives are online. This requires a minimum of (N/2 + 1) [Quorum](https://github.com/minio/dsync#lock-process) hard drives to create new objects.
+
+For example, an 8-node distributed Minio configuration with one hard drive per node will continue serving files even if as many as 4 hard drives are offline. This configuration requires at least 5 hard drives online to create new objects.
 
 ### Limits
 
-As with Minio in stand-alone mode, distributed Minio has a per tenant limit of minimum 2 and maximum 32 servers. There are no limits on number of disks shared across these servers. If you need a multiple tenant setup, you can easily spin up multiple Minio instances managed by orchestration tools like Kubernetes.
+Distributed Minio supports 2-32 servers per tenant. There are no limits on the number of hard drives shared across these servers and nor does Minio impose a limit to the number of hard drives used. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.
```

Comment:
Suggest rewording to something like this

`Distributed Minio requires at least 2 nodes and supports upto 32 nodes. There are no limits on the number of drives on each server, but total number of drives in the deployment should be even. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.`